projects/plugins/jetpack/class.jetpack.php

```diff
@@ -2232,6 +2232,15 @@ public function filter_default_modules( $modules ) {
 			}
 		}
 
+		// Special case to convert block setting to a block module.
+		$block_key = array_search( 'blocks', $modules, true );
+		if ( $block_key ) { // Only care if 'blocks' made it through the previous filters.
```

Comment:
We still need to process if it's at index 0.
```suggestion
		if ( $block_key !== false ) { // Only care if 'blocks' made it through the previous filters.
```